src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackageExtractionContext.cs

```diff
@@ -1,12 +1,14 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using NuGet.Common;
 using NuGet.Packaging.PackageExtraction;
 
 namespace NuGet.Packaging
 {
     public class PackageExtractionContext
     {
+        public ILogger Logger { get; set; }
```

Comment:
Good one, at first I didn't have this set in the constructor.

---
In reply to: [59911678](https://github.com/NuGet/NuGet.Client/pull/504#discussion_r59911678) [](ancestors = 59911678)